dashboard/src/t5gweb/ui.py

```diff
@@ -306,10 +309,11 @@ def trends():
     the previous quarter and creates report
     """
     cfg = set_cfg()
+    cards = redis_get("cards")
     return render_template(
         "ui/updates.html",
-        now=redis_get("timestamp"),
-        new_comments=get_trending_cards(),
+        now=get_timestamp(),
+        new_comments=get_trending_cards(cards),
```

Comment:
`get_trending_cards()` needs to be updated in the same manner as `get_new_comments()`

Reply:
the linter calls out load_data as well, but can we just delete load_data altogether now? I don't think its called anywhere anymore

Reply:
yeah @dcritch , deleted that function